src/components/duelling-picklist/duelling-picklist.stories.tsx

```diff
@@ -323,9 +473,9 @@ export const Grouped: Story = () => {
 Grouped.storyName = "Grouped";
 
 export const InDialog: Story = () => {
-  const [isDialogOpen, setIsDialogOpen] = useState(false);
+  const [isDialogOpen, setIsDialogOpen] = useState(defaultOpenState);
   return (
-    <>
+    <Box {...(defaultOpenState ? { height: 900 } : {})}>
```

Comment:
**suggestion(non-blocking)**: I noticed the decorator that added this wrapper was removed. I'm curious if it might not have had the intended effect. What are your thoughts on reintroducing a decorator specifically for the story? This would ensure the Box doesn’t appear in the story’s source code in the docs, which could potentially reduce confusion for readers.

```tsx
export const InDialog: Story = () => {};

InDialog.storyName = "In Dialog";
InDialog.decorators = [
  (InDialogStory) => (
    <Box width="100%" height="900px">
      <InDialogStory />
    </Box>
  ),
];
InDialog.parameters = {
  themeProvider: {
    chromatic: {
      theme: "sage", // tells Chromatic to render the story using the 'sage' theme only.
    },
  },
};
```

If this doesn't achieve the desired outcome either, I'm happy to continue the current approach.